.github/workflows/build-release.yml

```diff
@@ -111,3 +111,22 @@ jobs:
           files: |
             build/flow.phar
             build/flow.phar.asc
+
+      - name: "Get version"
+        id: get_version
+        run: |
+          echo "version=${GITHUB_REF/refs\/tags\//}" >> $GITHUB_OUTPUT
+
+      - name: "Update Homebrew formula"
+        uses: mislav/bump-homebrew-formula-action@v3
+        with:
+          formula-name: flow-php
+          formula-path: src/tools/homebrew/Formula/flow-php.rb
+          homebrew-tap: flow-php/flow
+          download-url: https://github.com/flow-php/flow/releases/download/${{ steps.get_version.outputs.version }}/flow.phar
+          commit-message: |
+            Update Homebrew TAP formula: {{formulaName}} to version: {{version}}
+
+            Created by https://github.com/mislav/bump-homebrew-formula-action
+        env:
+          COMMITTER_TOKEN: ${{ secrets.HOMEBREW_TAP_TOKEN }}
```

Comment:
can you rename into `${{ secrets.ACCESS_TOKEN }}` - its the same token automation is using for updating changelogs

Reply:
Its this one: 

<img width="573" alt="image" src="https://github.com/flow-php/flow/assets/1921950/4dd24573-c3af-4f08-bcf3-a733cc5ad27f">